org.metaborg.spoofax.meta.core/src/main/resources/org/metaborg/spoofax/meta/core/generator/general/langspec/metaborg.yaml

```diff
@@ -14,10 +14,6 @@ dependencies:
 {{#analysisNabl2}}  - org.metaborg:org.metaborg.meta.nabl2.shared:${metaborgVersion}
   - org.metaborg:org.metaborg.meta.nabl2.runtime:${metaborgVersion}
 {{/analysisNabl2}}
-pardonedLanguages:
```

Comment:
Why was this removed? These languages give false positives that should be pardoned.